examples/src/main/java/tachyon/examples/JournalCrashTest.java

```diff
@@ -62,6 +60,10 @@
      * Keep creating and renaming file.
      */
     CREATE_RENAME_FILE,
+    /**
+     * Keep creating empty raw table.
+     */
+    CREATE_TABLE,
```

Comment:
Updated, thanks (unfortunately it was addressed in part 1, so we won't see the change here)